index.bs

```diff
@@ -5930,6 +5930,17 @@ H(z) = \frac{\frac{b_0}{a_0} + \frac{b_1}{a_0}z^{-1} + \frac{b_2}{a_0}z^{-2}}
 $$
 </pre>
 
+This <a href="https://en.wikipedia.org/wiki/Infinite_impulse_response#Transfer_function_derivation">
+is equivalent</a> to the following time-domain equation:
```

Comment:
I don't think this link is helpful.  If you're going to implement the filters, you have to know a lot already.  And since we never talk about the transfer function again, we could just delete the stuff about the transfer function and just give the time-domain difference equation.

Reply:
I mean, I was implementing the filters and didn't know this :smile: